live-examples/js-examples/string/string-iterator.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">var str = "The quick red fox jumped over the lazy dog's back.";
+
+  var iterator = str[Symbol.iterator]();
```

Comment:
Again `const` might be a better fit here.

Reply:
This line and subsequent lines is getting 2 spaces of extra indentation that shouldn't be there.